src/components/dialogs/two-windings-transformer/two-windings-transformer-creation-dialog.js

```diff
@@ -759,6 +764,58 @@ const TwoWindingsTransformerCreationDialog = ({
                 isFormValid = false;
                 tabWithErrorList.push(DialogTab.RATIO_TAP_TAB);
             }
+
+            if (ratioTapRows.length > 1) {
```

Comment:
You could factorize this code a lot

Reply:
i extract a function to check both tables